NeuroID/NeuroIDClass/NeuroID.swift

```diff
@@ -110,7 +110,7 @@ public enum NeuroID {
         _ = NIDConfigService { success in
             if success {
                 if (NIDConfigService.nidConfigCache.callInProgress) {
-                    callObserver = NIDCallStatusObserver()
+                    callObserver = NIDCallStatusObserver(NIDConfigService.nidConfigCache.callInProgress)
```

Comment:
Not sure I follow why passing in the callInProgress flag is necessary as it will always be `true` here, no?

Reply:
oh, yup you are right